index.ts

```diff
@@ -298,16 +298,37 @@ collect.set('isEditingFile', [
 	'https://github.com/sindresorhus/refined-github/edit/ghe-injection/source/background.ts',
 ]);
 
+export const hasFileEditor = (url: URL | HTMLAnchorElement | Location = location): boolean => isEditingFile(url) || isNewFile(url) || isDeletingFile(url);
+collect.set('hasFileEditor', [
```

Comment:
You can also mark them as `combinedTestOnly` to avoid having to list the URLs. But it's fine either way